components/devtools/actors/browsing_context.rs

```diff
@@ -298,7 +298,8 @@ impl BrowsingContextActor {
         if let Some(p) = pipeline {
             self.active_pipeline.set(p);
         }
-        *self.url.borrow_mut() = url.as_str().to_owned();
+        // *self.url.borrow_mut() = url.as_str().to_owned();
```

Comment:
Perhaps you could remove it entirely instead of just commenting it out

Reply:
I have removed the comments.